test/testmetrics/metrics-hpa/01-assert.yaml

```diff
@@ -13,3 +13,14 @@ spec:
     metadata:
       labels:
         component: podtato-head-entry
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+collectors:
+  - type: pod
+    selector: app=podtato-head
+  - command: |
+      kubectl logs -l app=metrics-operator -n $NAMESPACE
```

Comment:
Hey @rizul2108 thanks for the work so far! One thing that wont work here is that metrics operator is installed in 'keptn-system' namespace not in $NAMESPACE, also you may want to use this label instead: control-plane=lifecycle-operator